src/nidigital/examples/burst_with_start_trigger_files/burst_with_start_trigger.digiproj

```diff
@@ -0,0 +1,38 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
I tried to open the project in DPE and got this error:
![image](https://user-images.githubusercontent.com/48041236/105077464-2c2e9300-5a52-11eb-8e49-7199334ab007.png)

Reply:
I suspect you were trying to follow the snake_case for file names used in `do_nothing` example (which BTW should be deleted since we now have real examples soon), but I'm starting to think that's a mistake. I think we're better off following the file name format provided by DPE, for support files. The actual example script file can still use snake_case.

Reply:
Yeah, I renamed the file to follow the snake_case because everything else seems to follow the snake_case.

If I stick with the original name for the project, I have to fix something in the build process. Because it cannot deal with file names with spaces in it.

Reply:
>The actual example script file can still use snake_case.

Here you left the `.digiproj` in snake case form. Is that because the original name had spaces?
Everything is in a folder and the folder has no spaces so I'd assume we can copy that fine at build time.

Reply:
I left the digiproj in snake case in part to match the name of the example and in part because a makefile in the build process is not friendly to file names with spaces in it.